ocs_ci/utility/utils.py

```diff
@@ -3106,6 +3106,35 @@ def skipif_upgraded_from(version_list):
         return False
 
 
+def skipif_no_kms():
+    """
+    This function evaluates the condition to skip a test if the cluster
+    doesn't have configured KMS encryption.
+
+    Return:
+        (bool): True if test needs to be skipped else False
+
+    """
+    from ocs_ci.ocs.resources import storage_cluster
+
+    cluster = storage_cluster.get_storage_cluster()
+    log.info("Checking if StorageCluster has configured KMS encryption")
+    try:
+        resource = cluster.get()["items"][0]
```

Comment:
@fbalak , can we make this kms check as an utility function something like `is_kms_enabled()` , so that even others can use it for checks.